doc/modules/cassandra/pages/operating/bulk_loading.adoc

```diff
@@ -80,13 +80,20 @@ The following options are supported, with `-d,--nodes <initial hosts>` required:
 -d,--nodes <initial hosts>                                   Required.
                                                              Try to connect to these hosts (comma separated) initially for ring information
 
+-e,--entire-sstable-throttle <throttle>                      Entire SSTable throttle
+                                                             speed in MiB/s (default 24MiB/s).
```

Comment:
Talked offline. Although the document says 24 MiB/s, the default is actually unlimited. I feel it makes sense with unlimited for the tool

Reply:
yes, this is correct for the bulk loading. I will adjust it

Reply:
Agreed, the change was triggered by the new doc and not the opposite - those were unlimited in the code for the tool for the 4 options